### ISSUE TITLE
what is wrong with netspeed gnome extension?

### ISSUE THREAD

**Issue by nazar2sfive** [clearlinux/distribution#1324]
![image](https://user-images.githubusercontent.com/12841320/66409250-0bcef680-ea10-11e9-86bf-fc542c8c7a76.png)

```
android@clr/ $ sudo swupd update
Password: 
Update started
Version on server (31230) is not newer than system version (31230)
Update complete - System already up-to-date at version 31230
```

**Comment by nsgnn** [comment]
If you press ``alt + f2`` then type ``lg`` in, you can see the looking glass window for gnome. If you go over to extensions, you can view what the error is.

**Comment by nazar2sfive** [comment]
![image](https://user-images.githubusercontent.com/12841320/66585143-102d1800-eba6-11e9-9852-ad95dacdcece.png)

**Comment by mrkz** [comment]
This is an extension issue, solved with the gnome 3.34 support merged in https://github.com/hedayaty/NetSpeed/pull/111

If still having this issue, please consider reporting in the extension github repository instead.

Thanks for your issue report.

**Comment by nazar2sfive** [comment]
![image](https://user-images.githubusercontent.com/12841320/66890593-5e0ea980-f006-11e9-81a3-a9249e382915.png)
